mlflow/models/evaluation/base.py

```diff
@@ -589,7 +585,7 @@ def check_nesting_config_dict(_evaluator_name_list, _evaluator_name_to_conf_map)
 _last_failed_evaluator = None
 
 
-def _get_last_failed_evaluator():
+def get_last_failed_evaluator():
```

Comment:
Can we make this internal again? https://github.com/mlflow/mlflow/pull/5256/files#r783315623